docs/setting-up/client/postgresql.md

```diff
@@ -245,10 +245,24 @@ If your PostgreSQL instance is configured to use TLS, click on the *Use TLS for
 !!! hint alert alert-success "Note"
     For TLS connection to work SSL needs to be configured in your PostgreSQL instance. Make sure SSL is enabled in the server configuration file `postgresql.conf`, and that hosts are allowed to connect in the client authentication configuration file `pg_hba.conf`. (See PostgreSQL documentation on [Secure TCP/IP Connections with SSL].)
 
+### Auto-discovery limit
+
+PMM 2.41.0 introduces limit for **Auto-discovery** in PostgreSQL, a feature that dynamically discovers all databases in your PostgreSQL instance. 
+
+Limiting **Auto-discovery** reduces connections and prevents high CPU and RAM usage caused by multiple databases.
```

Comment:
it also lead to collecting less metrics that could be collected from non-primary DB. @Nailya can we provide list of panels that doesn't show anything or works incorrectly?